nexteuropa_newsroom.context.inc

```diff
@@ -87,7 +87,7 @@ function nexteuropa_newsroom_context_default_contexts() {
   $context->conditions = array(
     'path' => array(
       'values' => array(
-        'newsroom/*' => 'newsroom/*',
+        'newsroom*' => 'newsroom*',
```

Comment:
I think this may be a bit too generic. Why not  split up the path's in multiple entries:
newsroom/*
newsroom